rxjava/src/androidTest/java/com/mindera/skeletoid/rxjava/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package com.mindera.skeletoid.rxjava;
+
+import android.content.Context;
+import android.support.test.InstrumentationRegistry;
+import android.support.test.runner.AndroidJUnit4;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.*;
```

Comment:
I know this is a dummy test but can we not have wildcard imports and convert it to kotlin?